tket/tests/test_PhaseGadget.cpp

```diff
@@ -132,7 +132,7 @@ SCENARIO("Full optimise_via_PhaseGadget") {
     const auto s0 = tket_sim::get_unitary(circ);
     Transforms::optimise_via_PhaseGadget(CXConfigType::Tree).apply(circ);
     REQUIRE(circ.count_gates(OpType::CX) == 12);
-    REQUIRE(circ.depth() == 13);
+    REQUIRE(circ.depth() == 14);
```

Comment:
What's causing the extra depth?

Reply:
This is because a TK1 gate gets commuted through a CX, whereas before it wasn't. In this particular case this impedes a previous merger of two TK1 gates. I don't think it's a big deal, just bad luck.

Reply:
Can we change the test from `== 14` to `<= 14`, so that future improvements don't fail it?